### PR TITLE
feat: Album document tab screen with new API

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [MOPLAT-480]

https://user-images.githubusercontent.com/17421923/200837896-42bbf12d-1a3d-496b-8dcd-270a1b4b759e.mov

### PR Checklist

- [x] I tested my changes on **iOS** and **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [x] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-480]: https://artsyproduct.atlassian.net/browse/MOPLAT-480?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/screens/HomeTabs/Albums/AlbumTabs/AlbumDocuments.tsx [hunk 1]
```diff
@@ -1,5 +1,66 @@
...
+export const albumDocumentsQuery = graphql`
+  query AlbumDocumentsQuery($partnerID: String!, $documentIDs: [String]) {
+    partner(id: $partnerID) {
+      documentsConnection(first: 100, documentIDs: $documentIDs) {
+        edges {
+          node {
+            internalID
+            title
+            size
+            publicUrl
```
Comment (dblandin) [inline src/app/screens/HomeTabs/Albums/AlbumTabs/AlbumDocuments.tsx:60]: **suggestion**: Remember to use the preferred fields of `publicURL` and `filesize` here so we can remove the deprecated `size` and `publicUrl` fields soon. 👍🏼

**Review by dblandin — APPROVED** [review]
Nice! 🥳

File: src/app/screens/HomeTabs/Albums/AlbumTabs/AlbumDocuments.tsx (hunk 1 above)
Comment (rajsam003) [reply src/app/screens/HomeTabs/Albums/AlbumTabs/AlbumDocuments.tsx:60]: Oh yes. I totally forgot. Thanks

**Review by pvinis — CHANGES_REQUESTED** [review]
nice. can we move the installs tab in the middle, so its consistent with the ShowTabs?

**Review by pvinis — APPROVED** [review]

**Pull request merged**